libs/ui-lib/lib/ocm/components/clusters/Clusters.tsx

```diff
@@ -33,9 +33,8 @@ import {
   selectClusterTableRows,
 } from '../../store/slices/clusters/selectors';
 
-type ClustersProps = RouteComponentProps;
-
-const Clusters: React.FC<ClustersProps> = ({ history }) => {
+const Clusters = () => {
+  const history = useHistory();
```

Comment:
If I remember correctly same functionality is possible to be achieved by using useLocation and useParams